switch_rgb.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+# Randomly sets a new preset
+# 1st arg is model name. 2nd argument -d to disable.
+
+if [ "$2" = "-d" ]
+then
+  echo "rgb disabled"
+  msi-perkeyrgb -m $1 $2
+  exit 0
+fi
+
+presets=(
+    aqua
+    chakra
+    default
+    disco
+    drain
+    freeway
+    plain
+    rainbow-split
+    roulette
+)
+
+selection=${presets[((RANDOM % 10+1))]}
```

Comment:
Might I suggest this?
```suggestion
selection=${presets[(( RANDOM % ${#presets[@]}+1 ))]}
```
This way the length isn't fixed to 10 in case more presets are added to the list. Also there are only 9 presets in the list and you are remaindering 10, so every once and a while there is a miss.

Reply:
Thanks! indeed